terminus-op/values.yaml

```diff
@@ -0,0 +1,47 @@
+replicaCount: 1
+
+statefulset:
+  port: 6363
+  image:
+    name: terminusdb/terminusdb-server
+    pullPolicy: Always
+    tag: ""
```

Comment:
Shouldn't we make the latest version the default tag?

Reply:
I am not using the tag anywhere. I'd like to know if you want to configure it.